langchain4j-core/src/test/java/dev/langchain4j/rag/DefaultRetrievalAugmentorTest.java

```diff
@@ -32,14 +37,31 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoInteractions;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
 
 class DefaultRetrievalAugmentorTest {
 
+    private static MockedStatic<LoggerFactory> loggerFactoryMock;
+
+    @BeforeAll
+    static void mockLogger() {
+        loggerFactoryMock = mockStatic(LoggerFactory.class);
+        Logger logger = mock(Logger.class);
+        when(LoggerFactory.getLogger(DefaultRetrievalAugmentor.class)).thenReturn(logger);
+        when(logger.isTraceEnabled()).thenReturn(true);
```

Comment:
Make sure that static returns `logger.isTraceEnabled() == true`. Doesn't seems to be the perfect fix, but increase code coverage back.